pkgs/development/python-modules/flupy/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  python,
+  fetchPypi,
+  setuptools,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "flupy";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Ekh6AI6XRM010PbqPPoG9LKyfLE4v1fQeI9cJuV6/mk=";
+  };
+
+  build-system = [ setuptools ];
+
+  buildInputs = [ typing-extensions ];
+
```

Comment:
Could you enable tests with `pythonImportsCheck` and `pytestCheckHook`?

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
Most tests for this project require external services and big downloads. They're not good for the CI.